demo/src/features/mesh/plugin.rs

```diff
@@ -32,16 +32,21 @@ impl RendererPlugin for MeshRendererPlugin {
         _upload: &mut RafxTransferUpload,
     ) -> RafxResult<()> {
         render_resources.insert(ShadowMapResource::default());
+        render_resources.insert(MeshRenderNodeSet::default());
+
+        render_resources.ensure_exists_or_add_default::<StaticVisibilityNodeSet>();
```

Comment:
the ensure_exists_or_add_default call is so that if two plugins require the same thing to be created, they can both call this and not stomp on each other